src/main/java/com/adyen/model/balanceplatform/AccountHolderCapability.java

```diff
@@ -121,7 +120,7 @@ public AllowedLevelEnum read(final JsonReader jsonReader) throws IOException {
 
   public static final String SERIALIZED_NAME_PROBLEMS = "problems";
   @SerializedName(SERIALIZED_NAME_PROBLEMS)
-  private List<CapabilityProblem> problems = null;
+  private List<Object> problems = null;
```

Comment:
The `CapabilityProblem` model is in LEM now, we need to review it's usage across services.

Reply:
Noted. We will add this as a known caviat ✅